NGitLab.Mock/Clients/UserClient.cs

```diff
@@ -158,8 +160,10 @@ public Models.User Update(int id, UserUpsert userUpsert)
             var user = Server.Users.GetById(id);
             if (user != null)
             {
-                user.Name = userUpsert.Name;
-                user.Email = userUpsert.Email;
+                // user.UserName = userUpsert.Username ?? user.UserName; // TODO
```

Comment:
Is there an reason of why the username can't be updated like the others?

Reply:
@Toa741 `NGitLab.Mock.User.UserName` is not settable. I assumed there is a reason for this, I just didn't have the time or need to investigate right now.